corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -186,13 +186,8 @@ def get_form_data_schema(request, domain, form_unique_id):
     """
     data = []
 
-    try:
-        form, app = Form.get_form(form_unique_id, and_app=True)
-    except ResourceConflict:
-        raise Http404()
-
-    if app.domain != domain:
-        raise Http404()
```

Comment:
this check might be still a good to have if someone tried to access a form and app from a different domain, even by mistake.

Reply:
`get_app` does this check: https://github.com/dimagi/commcare-hq/blob/ff388dfd6ad3e8e476126369270274c09ad5b9a2/corehq/apps/app_manager/dbaccessors.py#L234-L235